packages/language-server/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@prisma/language-server",
   "description": "Implementation of a language server in node.",
   "displayName": "Prisma Language Server",
-  "version": "2.0.1",
+  "version": "2.0.9",
```

Comment:
We since have 2.0.13 insider extension released but I guess CI will take care of releasing the correct language server version.